src/blockchain/projects.js

```diff
@@ -140,7 +140,7 @@ const projects = (app, liquidPledging) => {
     }
 
     if (data.length > 1) {
-      logger.info('more then 1 campaign with the same projectId found: ', data);
+      logger.info('more then 1 campaign with the same projectId found: ', data.length);
```

Comment:
Why statements like this have been changed?

Reply:
> Why statements like this have been changed?

The data was too large an I thought it's useless, because when the test codes reached to this line, technically all previous logs disapears and we misssed the, I think it's better instead of log all campaigns we log the length of that,
(campaigns and milestones have base64 logo so logging them make terminal so messy

Reply:
Replace it with something like this: `data.map(({_id, projectId, title}) => ({ _id, projectId, title }));`